src/eval/tests/mod.rs

```diff
@@ -371,14 +415,14 @@ fn evaluate_num_equal() {
     {
         let expr = "(= 5 5)";
 
-        let expected = s.t();
+        let expected = lurk_sym_ptr!(s, t);
         let terminal = s.get_cont_terminal();
         test_aux::<Coproc<Fr>>(s, expr, Some(expected), None, Some(terminal), None, 3, None);
     }
     {
         let expr = "(= 5 6)";
 
-        let expected = s.nil();
```

Comment:
I would actually recommend not changing the handling of `t` and `nil` in particular, since we anticipate a change to representation such that getting intended values in all context may not entail symbols.

Reply:
The change was pretty much syntactical (with a macro), so I don't think we'll be in a much worse (if worse at all) situation when those changes happen